src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -1312,7 +1302,20 @@ public void removePolicyMonitor(UUID id) {
 
   void handleMonitorPolicyEvent(MonitorPolicyEvent event) {
     log.info("Handling monitor policy event={}", event);
-    refreshBoundPolicyMonitorsForTenant(event.getTenantId());
+
+    String tenantId = event.getTenantId();
+    UUID policyId = event.getPolicyId();
+    UUID monitorId = event.getMonitorId();
+
+    // get effective policies
+    List<UUID> activePolicies = policyApi.getEffectiveMonitorPolicyIdsForTenant(event.getTenantId(), false);
```

Comment:
nit, since you have the variable
```suggestion
    List<UUID> activePolicies = policyApi.getEffectiveMonitorPolicyIdsForTenant(tenantId, false);
```